src/ClientData/include/ClientData/TimerData.h

```diff
@@ -66,6 +66,44 @@ class TimerData final {
 
   void UpdateMaxDepth(uint32_t depth) { max_depth_ = std::max(max_depth_, depth); }
 
+  [[nodiscard]] const orbit_client_protos::TimerInfo* GetFirstAfterTime(uint64_t time,
```

Comment:
Nit: Would this be an opportunity to rename the methods to `GetFirstAfterStartTime`? I find it confusing that we don't specify if we're using start- or end-times for such operations, but maybe that's just me.

Reply:
done.